includes/table/class-followers.php

```diff
@@ -192,13 +205,21 @@ public function prepare_items() {
 		);
 
 		foreach ( $followers as $follower ) {
-			$actor         = Actors::get_actor( $follower );
+			$actor     = Actors::get_actor( $follower->ID );
```

Comment:
```suggestion
			$actor     = Actors::get_actor( $follower );
```
If we pass the whole object, `get_actor()` doesn't need to grab it again.